specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,1955 @@
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "number": {
+                                "$ref": "#/definitions/PortNumber"
+                            },
+                            "protocol": {
+                                "type": "string",
+                                "enum": [
+                                    "TCP",
+                                    "UDP",
+                                    "*"
+                                ]
+                            },
+                            "allowedSourceAddressPrefix": {
+                                "type": "string"
+                            },
+                            "maxRequestAccessDuration": {
+                                "type": "string",
+                                "description": "Maximum duration requests can be made for. In ISO 8601 duration format. Minimum 5 minutes, maximum 1 day"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "JitNetworkAccessRequest": {
+            "type": "object",
+            "required": [
+                "requestor",
+                "startTimeUtc",
+                "virtualMachines"
+            ],
+            "properties": {
+                "virtualMachines": {
+                    "type": "array",
+                    "items": {
+                        "required": [
+                            "id",
+                            "ports"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "string",
+                                "description": "Resource ID of the virtual machine that is linked to this policy"
+                            },
+                            "ports": {
+                                "type": "array",
+                                "description": "The ports that were opened for the virtual machine",
+                                "items": {
+                                    "required": [
+                                        "allowedSourceAddressPrefix",
+                                        "endTimeUtc",
+                                        "number",
+                                        "status",
+                                        "statusReason"
+                                    ],
+                                    "type": "object",
+                                    "properties": {
+                                        "number": {
+                                            "$ref": "#/definitions/PortNumber"
+                                        },
+                                        "allowedSourceAddressPrefix": {
+                                            "type": "string",
+  
```

Comment:
This extension should not be applied for subscriptionId. We want subscriptionId to be a property on the client.